model_compiler/python/nnir_to_clib.py

```diff
@@ -298,8 +298,8 @@ def generateCMakeExtras(graph, outputFolder):
 endif()
 
 find_package(OpenCV QUIET)
-include_directories (/opt/rocm/include/miviisionx ../)
-link_directories    (/opt/rocm/lib)
+include_directories (${ROCM_PATH}/include ${ROCM_PATH}/include/mivisionx ../)
```

Comment:
@hansely can you confirm why we have `../` as one of the dirs? is this typo or can we fix this with a full link?

Reply:
@kiritigowda sorry that is a typo. Would you be able to remove it in this PR?